bower.json

```diff
@@ -19,17 +19,35 @@
   },
   "main": "paper-toast.html",
   "dependencies": {
-    "iron-a11y-announcer": "PolymerElements/iron-a11y-announcer#^1.0.0",
-    "iron-overlay-behavior": "PolymerElements/iron-overlay-behavior#^1.0.9",
-    "iron-fit-behavior": "PolymerElements/iron-fit-behavior#^1.1.0",
-    "polymer": "Polymer/polymer#^1.5.0"
+    "iron-a11y-announcer": "PolymerElements/iron-a11y-announcer#2.0-preview",
+    "iron-overlay-behavior": "PolymerElements/iron-overlay-behavior#2.0-preview",
+    "polymer": "Polymer/polymer#^2.0.0-rc.1"
```

Comment:
paper-toast requires iron-fit-behavior 1.1.0 or superior.
iron-overlay-behavior 1.x does not enforce this (it has dependency on iron-fit-behavior 1.0.0), so the 1.x variant of paper-toast needs to specify that dependency.
Iron-overlay-behavior 2.x already satisfies paper-toast dependency on iron-fit-behavior.
If it helps it to be clearer, I can add iron-fit-behavior dependency also in the main variant